src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -5626,6 +5626,31 @@ void M(int x)
         M2(ref [|(y = ref x)|]);
     }
     void M2(ref int p) { }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        [WorkItem(27949, "https://github.com/dotnet/roslyn/issues/27949")]
+        public async Task TestWhitespaceSpanInAssignment()
```

Comment:
❔ Out of curiosity, many of the new tests failed prior to the change?

Reply:
All four added tests were failing before the fix.